moments/react-native/storyly-moments-react-native/android/src/main/java/com/appsamurai/storyly/moments/reactnative/STStorylyMomentsModule.kt

```diff
@@ -29,8 +37,47 @@ class STStorylyMomentsModule(reactContext: ReactApplicationContext) : ReactConte
           momentToken = token,
           userPayload = userPayload
         )
-      )
-//      storylyMomentsManager?.momentsListener = storylyMomentsListener
+      ).apply {
+        momentsListener = object : MomentsListener {
+          override fun storylyMomentsEvent(
+            event: StorylyMomentsEvent,
+            momentsStoryGroup: MomentsStoryGroup?,
+            stories: List<MomentsStory>?
+          ) {
+            sendEvent(
+              EVENT_STORYLY_MOMENTS_EVENT,
+              Arguments.createMap().also { eventMap ->
+                eventMap.putString("eventName", event.name)
+                eventMap.putMap("storyGroup", momentsStoryGroup?.let { createMomentsStoryGroup(it) })
+                eventMap.putArray("stories", stories?.let { createMomentsStoryList(stories) })
+              })
+          }
+
+          override fun onOpenCreateStory(isDirectMediaUpload: Boolean) {
+            sendEvent(
+              EVENT_STORYLY_MOMENTS_OPEN_STORY_CREATE,
+              Arguments.createMap().also { eventMap ->
+                eventMap.putBoolean("isDirectMediaUploaded", isDirectMediaUpload)
+              })
+          }
+
+          override fun onOpenMyStory() {
+            sendEvent(EVENT_STORYLY_MOMENTS_OPEN_MY_STORY, Arguments.createMap())
+          }
+
+          override fun onUserStoriesLoaded(momentsStoryGroup: MomentsStoryGroup?) {
+            sendEvent(EVENT_STORYLY_MOMENTS_USER_STORIES_LOADED, Arguments.createMap().also { eventMap ->
+              eventMap.putMap("storyGroup", momentsStoryGroup?.let { createMomentsStoryGroup(it) })
```

Comment:
@levent-oral  for android, `onUserStoriesLoaded` event is not triggering. will test if there is any issue on the native SDK side?